Jint/Native/Function/FunctionInstance.cs

```diff
@@ -8,7 +8,7 @@ namespace Jint.Native.Function
 {
     public abstract class FunctionInstance : ObjectInstance, ICallable
     {
-        protected internal PropertyDescriptor _prototype;
+        protected internal PropertyDescriptor _prototypeDescriptor;
```

Comment:
Where can I find in the spec the explanation for this refactoring?

Reply:
This was just because `_prototype` needs to be a field for `ObjectInstance` for `set/getPrototypeOf` and this was hiding it